src/android/getPrefix.js

```diff
@@ -0,0 +1,16 @@
+const path = require('path');
+const semver = require('semver');
+
+module.exports = function getPrefix(config) {
+  const rnpkg = require(
+    path.join(config.folder, 'node_modules', 'react-native', 'package.json')
+  );
+
+  var prefix = 'patches/0.18';
+
+  if (semver.lt(rnpkg.version, '0.18.0')) {
+    prefix = 'patches/0.17';
+  }
+
+  return prefix;
```

Comment:
I'd prefer to keep it this way for the future versions. Ternary operator works fine in this case, but if in the next react-native version facebook will introduce something else, then we'll need to roll our back to the this one.